dll/cpl/input/settings_page.c

```diff
@@ -11,192 +11,341 @@
 #include "locale_list.h"
 #include "input_list.h"
 
+static INT s_nLeavesCount = 0;
```

Comment:
Isn't this somehow related to the number of items/"nodes" in the `_InputList` ?

Reply:
This is the alive leaf count. Aliveness can be changed anytime.
I don't want to manage it in `INPUT_LIST_NODE` structure.

Reply:
`InputList_GetAliveCount` function is added.

Reply:
`InputList_GetAliveCount` function is added.